R/utility_trips.R

```diff
@@ -33,6 +33,8 @@ make_od = function(oas, os_pois, grid, purpose, trip_purposes, zones, parameters
   os_pois = sf::st_join(os_pois, grid, join = sf::st_nearest_feature)
 
   # calculate weighting of each grid point  
+  browser()
+  os_pois
```

Comment:
Browser() used to debug objects at this point. Clear the issue from this: the column is lacking and that's due to a previous step that removed the classname column:

```
os_pois
Simple feature collection with 52137 features and 2 fields
Geometry type: POINT
Dimension:     XY
Bounding box:  xmin: 195991 ymin: 530348 xmax: 397421.6 ymax: 675769
Projected CRS: OSGB36 / British National Grid
First 10 features:
     ref_no grid_id                  geom
1  18218308   34397 POINT (287405 612972)
2  18219911   42816 POINT (309823 624984)
3  18219916   43515 POINT (304793 626017)
4  18219917   42097 POINT (303311 623852)
5  18219918   43870 POINT (304837 626270)
6  18219919   43155 POINT (302152 625568)
7  18219920   43155 POINT (302164 625422)
8  18219921   44222 POINT (302712 626908)
9  18220720   67719 POINT (316984 657013)
10 18220730   67280 POINT (319049 656495)
```